src/Middleware/LoginSessionMiddleware.php

```diff
@@ -31,7 +32,8 @@ public function process(HTTPRequest $request, callable $delegate)
 
         try {
             $loginSessionID = $request->getSession()->get($loginHandler->getSessionVariable());
-            $loginSession = LoginSession::get()->byID($loginSessionID);
+            /** @var LoginSession $loginSession */
+            $loginSession = DataObject::get_by_id(LoginSession::class, $loginSessionID);
```

Comment:
I'll suggest this (again)..
```suggestion
            $loginSession = LoginSession::get_by_id($loginSessionID);
```